### ISSUE TITLE
Default HOME as unique

### ISSUE THREAD

**Issue by alainroger** [JoomFish/joomfish-2.5#122]
Hi Everybody,

once again this topic is here. In latest version of JF-Future downloaded yesterday i have the same issue, when several languages are used...HOME isnot anymore unique and can't be saved in case of modification of home parameters :(
see issue #33

**Comment by klas** [comment]
Please try this revision https://github.com/JoomFish/jf-future/tree/2553533bd30c0820560f0fc5a0836eb5ecb537ef

My guess is contentelement loading could be broken after https://github.com/JoomFish/jf-future/commit/71c1e9bbd398cb1877dcc734a59b9707187164b5 and since necessary functions to work around menu issue are not executed

**Comment by alainroger** [comment]
Is it an older version than the existing one or a newer ?

**Comment by klas** [comment]
This is few revisions older than the last one

2013/1/16 alainroger notifications@github.com

**Comment by akempkens** [comment]
Klas, what part of the content element loading do you think is broken? The change is related to loading different translation object classes. The loader still loads the rights classes.

**Comment by klas** [comment]
can't test from where I am at the moment so this is just a guess, it could be that translaltionObjectMenu is not properly loaded and since unsetHomeTr is not run which would cause this error. Relates to transaltionObject subclasses loading, not content element itself.

**Comment by akempkens** [comment]
Ok I was able to debug that during the translation process the preHandler is called. The class is loaded correctly.

However during my investigations so far I figured out that the update of language information within the original table rows (the one you translate) seem to get out of synch. At least what the Joomla interface is showing. Even so a translation exists the Home menu still shows it is "default" and for "all" languages.

As soon as you edit the menu it switches to English only and starting at this point the described issues appear. I feel some more investigations are needed.
